src/quo2/components/tags/tags.cljs

```diff
@@ -151,16 +153,18 @@
                                                                 :viewPosition 0.5}))
                                             (when on-change
                                               (on-change id)))}
-                             label]])})])
+                             name]])})])
           [rn/view {:style {:flex-direction :row}}
-           (for [{:keys [label id resource]} data]
+           (for [{:keys [id name resource]} data]
```

Comment:
wha't the purpose if changing label to name? its better to avoid `name` because if shadowing clojure core